dotnet-desktop-guide/samples/snippets/visualbasic/VS_Snippets_Wpf/InvokePatternApp/VisualBasic/InvokePatternClient/Client.vb

```diff
@@ -149,7 +145,7 @@ Namespace InvokePatternSample
             ' a stack overflow. If you are attempting to obtain a specific 
             ' element at a lower level, you should start your search from the 
             ' application window or from a container at a lower level.
-            targetApp = _
```

Comment:
VB code needs the `_` to continue to the next line. Free continuation isn't allowed like it is in C#

Reply:
Hmm, Ctrl+K,D removed it, not me.